apps/portal/src/components/pages/MagicLinkPage.js

```diff
@@ -180,20 +188,28 @@ export default class MagicLinkPage extends React.Component {
 
         return (
             <form onSubmit={e => this.handleSubmit(e)}>
-                <section className='gh-portal-section'>
+                <section className='gh-portal-section gh-portal-otp'>
                     {/* @TODO: create different input component with updated design */}
-                    <InputField
```

Comment:
If we're switching away from `<InputField>` it would be good to clean up the [OTC-related stuff here](https://github.com/TryGhost/Ghost/blob/main/apps/portal/src/components/common/InputField.js#L131-L139)